site/ca_ES.all.json

```diff
@@ -1181,15 +1181,15 @@
     "other": "Preu"
   },
   "shopping_price_plan_note": {
-    "one": "Es {{ .Interval }} cada .Interval després que finalitzi la prova gratuïta de 24 hores.",
-    "other": "Es {{ .Interval }} cada .Interval després que finalitzi la prova gratuïta de {{ .Count }}"
+    "one": "Es {{.Interval}} cada .Interval després que finalitzi la prova gratuïta de 24 hores.",
```

Comment:
this has .Interval in there twice, with the second not in moustaches. not sure if thats intended or not?